demos/common/python/models/yolo.py

```diff
@@ -305,3 +314,92 @@ def sigmoid(x):
                 objects.append(Detection(x - width / 2, y - height / 2, x + width / 2, y + height / 2,
                                          confidence.item(), class_id.item()))
         return objects
+
+class YOLOF(YOLO):
+    class Params:
+        def __init__(self, num, num_objects, anchors):
+            self.num = num
+            self.coords = 4
+            self.classes = 80
+            self.topk = 1000
+            self.cells = int(np.sqrt(num_objects // num))
+            self.anchors = anchors
+
+    def _get_output_info(self):
+        num = 6
+        anchors = ANCHORS['YOLOF']
+
+        output_info = {}
+        for i, (name, layer) in enumerate(self.net.outputs.items()):
+            shape = layer.shape
+            yolo_params = self.Params(num, shape[1], anchors)
+            output_info[name] = (shape, yolo_params)
+        return output_info
+
+    def postprocess(self, outputs, meta):
+        detections = []
+
+        for layer_name in self.yolo_layer_params.keys():
+            out_blob = outputs[layer_name]
+            layer_params = self.yolo_layer_params[layer_name]
+            out_blob.shape = layer_params[0]
+            detections += self._parse_yolo_region(out_blob, meta['resized_shape'], layer_params[1], self.threshold)
+
+        detections = self._filter(detections, self.iou_threshold)
+
+        detections = self._resize_detections(detections, meta['original_shape'][1::-1], meta['resized_shape'][1::-1])
+
+        return clip_detections(detections, meta['original_shape'])
```

Comment:
Looks like there is no difference with `YOLO::postprocess` method. Why did you reimplement it?